deploy/docker/build.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+IMAGE=${1-heapster:canary}
```

Comment:
`${1-heapster:canary}` means `$1 if $1 is empty (or undefined, same thing) otherwise "heapster:canary"`. There is an alternate and more popular notation, `${1:-heapster:canary}`, which does the same.